grammar.js

```diff
@@ -825,7 +827,7 @@ module.exports = grammar({
 
     range_test: $ => seq($._in_operator, $._expression),
 
-    type_test: $ => seq($._is_operator, $._expression),
+    type_test: $ => seq($._is_operator, $._type),
```

Comment:
there is actually also a _is_operator in check_expression where we should also use a _type instead of _expression.

Reply:
good to know!